sru/__init__.py

```diff
@@ -5,7 +5,4 @@
 in "Simple Recurrent Units for Highly Parallelizable Recurrence."
 """
 from .version import __version__
-try:
-    from .sru_functional import *
-except:
-    from sru_functional import *
+from .modules import *
```

Comment:
in general i find it more robust to do: `from sru.modules import *`. Aside, seems like the try except was there for a reason, any reason to remove it?

Reply:
Thanks. I'll use `sru.modules`. The try except code was some legacy code that long time ago I or some people are not using the code as a library. so I added `from sru_functional import *` to support that. 
But it became a problem since many report issues are failures to compile the code and the try-except code hide the real error messages / stack trace information. so i decide to just make it a library-like import.